### PR TITLE
Adding prow to istio-ecosystem

### PR DESCRIPTION
**Pull request by icygalz**
The hook is returning green now! PR ready for review.

### CODE REVIEW

File: prow/plugins.yaml [hunk 1]
```diff
@@ -10,7 +10,9 @@ triggers:
   - istio/test-infra
   - istio/proxy
   - istio-releases/pipeline
+  - istio-ecosystem/authservice
   trusted_org: istio
+  trusted_org: istio-ecosystem
```
Comment (sebastienvas) [inline prow/plugins.yaml]: I don t think that s valid YAML:
add another section:
```YAML
- repos
  - istio-ecosystem/authservice
  trusted_org: istio-ecosystem
```

File: prow/plugins.yaml [hunk 2]
```diff
@@ -43,6 +45,7 @@ lgtm:
   - istio/test-infra
   - istio/proxy
   - istio-releases/pipeline
+  - istio-ecosystem/authservice
   trusted_team_for_sticky_lgtm: "Istio Hackers"
```
Comment (sebastienvas) [inline prow/plugins.yaml:51]: Same comment here you might want to create another section

File: prow/plugins.yaml (hunk 1 above)
Comment (icygalz) [reply prow/plugins.yaml]: fixed

File: prow/plugins.yaml (hunk 2 above)
Comment (icygalz) [reply prow/plugins.yaml:51]: fixed

**Comment by icygalz** [comment]
Webhook is green now!
/hold cancel

**Pull request merged**